code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -170,7 +171,7 @@
 	visible_message("<span class='notice'>The ore was swallowed whole!</span>")
 
 /mob/living/simple_animal/hostile/asteroid/goldgrub/proc/Burrow()//Begin the chase to kill the goldgrub in time
-	if(!alerted)
+	if(!alerted && will_burrow)
 		alerted = 1
```

Comment:
If you're wondering what this is about: The abandoned zoo has a goldgrub with `will_burrow` set to false - but since that var doesn't exist, attempting to load it will runtime. What this does to remedy this, is to set a var that does what you'd expect `will_burrow` to do - meaning that the goldgrub in the abandoned zoo won't be able to burrow away.